### PR TITLE
Fix various parts of the post

### PR DESCRIPTION
**Pull request by MTRNord**

### CODE REVIEW

**Review by tulir — APPROVED** [review]

File: content/blog/2024/09/2024-09-22.md [hunk 1]
```diff
@@ -263,6 +237,36 @@ Matrix SDK written in pure Dart.
 > * Discussion, suggestions and ideas are welcome in #libmatrix:rory.gay (Space: #mru-space:rory.gay)
 > * Got a cool project that you're working on and want to share, using LibMatrix? Be sure to let us know, we'd love to hear all about it!
 
+## Dept of Ops 🛠
```
Comment (sandhose) [inline content/blog/2024/09/2024-09-22.md:240]: That's not right is it?

Comment (MTRNord) [reply content/blog/2024/09/2024-09-22.md:240]: It is. The diff is just a bit weird. Matrix Utils is an ops tool (afaik?)  and the LibMatrix one was moved one level up to the SDKs

Comment (MTRNord) [reply content/blog/2024/09/2024-09-22.md:240]: Github just shows the wrong thing as being moved here

Comment (MTRNord) [reply content/blog/2024/09/2024-09-22.md:240]: (I seem to have missed the "Emma reported" part though...)

Comment (MTRNord) [reply content/blog/2024/09/2024-09-22.md:240]: Opened https://github.com/matrix-org/matrix.org/pull/2503 for that

Comment (sandhose) [reply content/blog/2024/09/2024-09-22.md:240]: > Matrix Utils is an ops tool (afaik?)  and the LibMatrix one was moved one level up to the SDKs

TIL!

**Pull request merged**